recipes/package_repo.rb

```diff
@@ -31,7 +31,7 @@
   when 'debian'
     apt_repository 'mongodb' do
       uri node['mongodb3']['package']['repo']['url']
-      distribution "#{node['lsb']['codename']}/mongodb-org/stable"
+      distribution "#{node['lsb']['codename']}/mongodb-org/3.2"
```

Comment:
Thank you for the wonderful PR.
btw, we have conflict PR for this line. #14 
I'm going to merge that and modifying the attribute value with `3.2`
Sounds good? ;-)
And also, I'll (may) try ... 
1. Testing whether this PR support the old mongodb version
2. Adding if condition to set the apt key value for each 3.0.x and 3.2.x (if new key is not working for 3.0.x)

Thank you.